### PR TITLE
Arabic Language Support

### PR DESCRIPTION
**Pull request by hazanberg**
My Arabic Translation for the App. This should be available for all the Arabic locales.
ar_DZ
ar_BH
ar_EG
ar_IQ
ar_JO
ar_KW
ar_LB
ar_LY
ar_MA
ar_OM
ar_QA
ar_SA
ar_SD  
ar_SY
ar_TN
ar_AE
ar_YE

### CODE REVIEW

**Comment by YaPSNapp** [comment]
Thank you very much for this PR. I will take a look at it soon and merge it for next version.
Did Sony finally translated trophies in Arabic language too ?

**Comment by hazanberg** [comment]
I live in the US and do not use Arabic that often. However, I heard that Sony has games released in the Arabic language. Their system does not have Arabic menu or trophies. But there are many mainstream games that are available in Arabic.
Getting the App translated to Arabic was just something that I was able to do to support you. I am a web designer/developer. If you need help with design or with web language coding, I am available.
You created the best PlayStation App on apple handheld devices. Maybe the next thing to do is to design one for iPad or iPad mini? No matter what your plan is, just continue on what you are doing, you have all the support from me! Good luck on your next step!

**Comment by YaPSNapp** [comment]
I have integrated the Arabic language in the upcoming version.
My question is regarding Arabic language common use ... As Arabic language user, is it something common to have apps translated in arabic ... but still in Left to Right language orientation ?
 ... It will make a lot of extra work to be able to display the app with a RTL orientation ... But I'm ready to do it if it is better like this :P

**Comment by YaPSNapp** [comment]
Hi,
I will release a private beta version of the app that will allow each contributor to test their localization.
If you want to participate, please send me an email to yapsnapp@drakfly.com.

File: Localization/ar_AR/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+
+"minuteUnits" = "دقيقة";
+"hourUnits" = "ساعة)";
+"dayUnits" = "يوم";
+"weekUnits" = "اسبوغ";
+"monthUnits" = "شهر";
+"yearUnits" = "سنة";
+//ERRORS
+"OP_AUTHFRIENDS_SERVER_ERROR" = "شبكة البلاي ستاشين غير متوفرة حالياً. نرجوا المحاولة لاحقاً.";
+"OP_AUTHFRIENDS_AUTHENTICATION_ERROR" = "اسم المستخدم او كلمة المرور غير صحيحة";
```
Comment (HNC-77) [inline Localization/ar_AR/Localizable.strings:24]: Salmantoma2@gmail.com

File: Localization/ar_AR/AppStore.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+• التروفي (نقوم بتحميلها ونوفرها لك عندما لا تكون متصلاً)
+• النصائح (الذي يضم طرق للحصول على البلاتين)
+• أكثر من حساب واحد (الأصدار الكامل فقط)
+• متابعة حالات اصدقائك واستلام رسائلهم من البلاي ستاشين (الاصدار الكامل فقط)
+• كتابة رسائل البلاي ستاشن من التطبيق (الأصدار الكامل فقط)
+• البحث عن التروفي
+• الدخول الأمن 
+• السرعة في التصفح
+
+اذا كان لديك حساب بلاي ستاشين آخر خاص بك. ليس هناك مشكلة، فالتطبيق يسمح لك بأضافة أكثر من حساب، وتحويل من حساب إلى اخر بكبسة زر.
```
Comment (HNC-77) [inline Localization/ar_AR/AppStore.txt:21]: HNC-77

File: Localization/ar_AR/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+"ERROR_FRIENDS_LIST_PARSING_XML" = "لم نستطع جمع قائمة اصدقائك من شبكة البلاي ستاشين. ارجوا المحاولة لاحقاً.";
+"ERROR_GAMES_LIST_PARSING_XML"="هناك خطأ في شبكة البلاي ستاشين. نرجوا المحاولة لاحقاً.";
+"UNKNOW_ERROR" = "حدث خطأ. نرجوا المحاولة مرة أخرى.";
+"NODATAENABLE" = "يجب عليك الأتصال بشبكة وايفاي او خلوي.";
+/** Settings Controller */
+"SettLanguage" = "اللغة";
+"SettMailTo" = "yapsnapp@drakfly.com";
+"SettMailSubject" = "YaPSNapp support (arabic)";
+"SettMailBody" = "<br/><br/><b>YaPSNapp الأصدار:</b>%@<br/><b>الجهاز:</b>%@<br/><b>OS:</b>%@<br/><b>UID</b>:%@";
+"SettNoEmail" = "لم يتم اعداد بريدك الألكتروني على جهازك. نرجوا منك الذهاب إلى الأعدادات وتفعيل بريدك الألكتروني.";
```
Comment (HNC-77) [inline Localization/ar_AR/Localizable.strings:36]: pppo7728@gmail.com

**Comment by HNC-77** [comment]
pppo7728@gmail.com

**Pull request merged**